google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java

```diff
@@ -644,6 +651,312 @@ public void testGetBlobFailure() {
     }
   }
 
+  private void verifyCreateBlobCapturedStream(ArgumentCaptor<ByteArrayInputStream> capturedStream)
+      throws IOException {
+    ByteArrayInputStream byteStream = capturedStream.getValue();
+    byte[] streamBytes = new byte[BLOB_CONTENT.length];
+    assertEquals(BLOB_CONTENT.length, byteStream.read(streamBytes));
+    assertArrayEquals(BLOB_CONTENT, streamBytes);
+    assertEquals(-1, byteStream.read(streamBytes));
+  }
+
+  @Test
+  public void testCreateBlob() throws IOException {
+    ArgumentCaptor<ByteArrayInputStream> capturedStream =
+        ArgumentCaptor.forClass(ByteArrayInputStream.class);
```

Comment:
I have noticed you have used  `ArgumentCaptor<ByteArrayInputStream> capturedStream = ArgumentCaptor.forClass(ByteArrayInputStream.class);`   at many places so declare it as globally to remove the duplication.

Reply:
Thanks for that comment. 
`capturedStream` is not a constant, it's a variable used by a dozen tests which is less than 10% of total tests. This variable becomes useless after one usage, so each test should recreate it before new use.

Using global variables is not a good practice. It's safer to use a local instance.